### PR TITLE
Initial labelmatch implementation

### PR DESCRIPTION
**Pull request by sandroacoelho**

### CODE REVIEW

**Comment by fsasaki** [comment]
This change has no issue related to it. @sandroacoelho , can you summarize in an issue what has been done? Also, one needs to update the documentation, I assume, to have the new parameter labelmatch documented? Putting @jnehring into the loop FYI.

**Comment by sandroacoelho** [comment]
#171

**Comment by fsasaki** [comment]
Thanks for referencing https://github.com/freme-project/freme-ner/issues/171 , Sandro. That issue shows the motivation of the change. It would be nice to also have the implementation briefly (bullet points are enough!) summarised and the documentation updated.

**Comment by sandroacoelho** [comment]
Hi @fsasaki. As you have mentioned, this issue is related with our Label Matching approach and contains

- A new endpoint to deal with Label Matching: /e-entity/freme-ner/labelmatch
- A spotter(string matching algorithm) + dictionary;

It is a very beginning implementation. We need to test the approach before document it (cc: @m1ci, @gustavo).

I will keep you updated.

Best,

**Comment by fsasaki** [comment]
Makes sense, thanks, Sandro.

2017-01-23 11:30 GMT+01:00 Sandro <notifications@github.com>:
